indico/modules/events/themes.yaml

```diff
@@ -47,3 +47,33 @@ definitions:
     template: weeks.html
     title: Indico Weeks View
     user_visible: true
+    user_settings:
+      colors:
+        caption: Contribution Colours
+        description: 'Colours used for different types of timetable entries'
+        type: MultipleItemsField
+        required: false
+        kwargs:
+          fields:
+            - id: entry
+              caption: Entry type
```

Comment:
Shouldn't we mention keywords here? Otherwise it's not clear what the entry type is!